app/src/main/java/com/example/a220402/RepoFragment.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.a220402
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.a220402.databinding.FragmentRepoBinding
+
+class RepoFragment : Fragment() {
+    private lateinit var repoAdapter: RepoAdapter
+    private var _binding : FragmentRepoBinding? = null
+    private val binding get() = _binding!!
```

Comment:
```suggestion
    private val binding get() = _binding ?: error("binding이 초기화 안 됐으니 어쩌구저쩌구 ~")
```

`!!`은 절대 null이 아니라고 컴파일러에게 알려주는 역할입니다(개발자가 컴파일러한테 `null 아니야 !!`라고 소리치는 모양이라나 뭐라나). 만약에, 만약에 불의의 사고로 _binding이 null이 되면 이 앱은 `NullPointerException`이라는 런타임 에러를 띄우며 죽습니다. 따라서 `?:` 연산자를 이용해 조금 더 안전하게 처리하면 어떨까 싶습니다 :-)